packages/records/lib/models/records.ts

```diff
@@ -22,6 +22,23 @@ dayjs.extend(utc);
 
 const BATCH_SIZE = 1000;
 
+export async function getRecordsCount({ connectionId, model }: { connectionId: number; model: string }): Promise<Result<number>> {
+    try {
+        const result = (await db(RECORDS_TABLE).count('* as object_count').where({ connection_id: connectionId, model, deleted_at: null }).select()) as {
+            object_count: number;
+        }[];
+
+        if (result.length === 0) {
+            return Ok(0);
+        }
+
+        return Ok(result[0]?.object_count ?? 0);
```

Comment:
`count` from Postgres are returned as bigInt which are interpreted by knex (the db library we are using) as string. More info https://knexjs.org/guide/query-builder.html#count
Typescript compiler is lying here since it is not necessary returning a `Result<number>` 😉 

I would do something like this and ensure it returns a number:
```
    const [result] = await db
       .from(RECORDS_TABLE).count('*')
       .where({
          connection_id: connectionId,
          model,
          deleted_at: null
       });
    return Ok(result ? Number(result) : 0);
```